src/test/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesRestartTest/upgradeFrom_1_27_1/config.xml

```diff
@@ -81,6 +81,14 @@
                 <periodSeconds>0</periodSeconds>
                 <successThreshold>0</successThreshold>
               </livenessProbe>
+              <readinessProbe>
```

Comment:
Please revert all these test resources hunks. They represent old configuration, they must not be modified in order to let tests pass.